node/localnode.go

```diff
@@ -400,9 +405,9 @@ func (localNode *LocalNode) findAddr(key []byte, tls bool) (string, []byte, []by
 	if tls == true {
 		if nodeData.TlsWebsocketDomain != "" {
 			host = nodeData.TlsWebsocketDomain
-			port = config.Parameters.HttpWssPort
+			port = uint16(nodeData.TlsWebsocketPort)
 		} else {
```

Comment:
It seems that unnecessary to `address, err := url.Parse(pred.Addr) && host := address.Hostname()`([L394](https://github.com/nknorg/nkn/blob/2492f04d6985a4b4c2c3dc5a45e55f1e61c3b3fd/node/localnode.go#L394) ~ [L402](https://github.com/nknorg/nkn/blob/2492f04d6985a4b4c2c3dc5a45e55f1e61c3b3fd/node/localnode.go#L402))  if `tls != true`

Reply:
You are right, fixed.